CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 ### Added
 
-- `Sum` for scalars like `Rint`, `Rfloat` and `Rcplx`, which accept `Iterator<Item = &Rtype>` [[#454]](https://github.com/extendr/extendr/pull/454) 
+- [**ndarray**] `TryFrom<&Robj>` for `ArrayView1<T>` and `ArrayView2<T>`, where `T` is `i32`, `f64`, `c64`, `Rint`, `Rfloat`, `Rcplx`, `Rstr`, `Rbool` [[#443]](https://github.com/extendr/extendr/pull/443)
+- `Sum` for scalars like `Rint`, `Rfloat` and `Rcplx`, which accept `Iterator<Item = &Rtype>` [[#454]](https://github.com/extendr/extendr/pull/454)
```

Comment:
I am not following, what is not needed?

Reply:
This line in the changelog about `Sum`. It was in the other PR.

Reply:
I think it is still correct. You see in diff two lines because I added `ndarray` atop `Sum`. And `Sum` was indeed merged as a separate PR before, I think it is worth mentioning here.